src/node_modules/app/utils.js

```diff
@@ -32,3 +33,23 @@ let lastId = 0;
 export function autoID () {
 	return `${IS_BROWSER ? 'client' : 'server'}${lastId++}`;
 }
+
+const createPropDefs = _.pipe([
+	_.mapWith(property => `	${property[0]}: ${property[1]};`)
+])
```

Comment:
`pipe` here is unnecessary (a leftover I suppose). Also I'd use desctructuring to be more explicit about what's going on:

```suggestion
const createPropDefs = _.mapWith(([field, EStype]) => ` ${field}: ${EStype};`);
```

you could also do

```js
import {joinWith} from '@svizzl/utils';

const createPropDefs = _.mapWith(_pipe([joinWith(': '), prepend(' ')]))
```